### DISCUSSION TITLE
New dev

### DISCUSSION THREAD

**Discussion by sp8cefox** [mikedilger/gossip#903] [Q&A] [answered]
Hi, I'm a 'new' dev in context of contributing to this github repo.
Do you guys have dev calls or how does communication work on this project?

**Comment by mikedilger** [comment, marked as answer]
We don't have calls or meetings.  Mostly we just communicate on github, but we used to have a telegram channel.

What you should do is open an issue to propose new code ideas, or open a pull request to submit code. That is the only way to be a developer on the project.
